suite-native/module-authorize-device/src/hooks/useInitiateThpConnection.ts

```diff
@@ -0,0 +1,17 @@
+import { useDispatch, useSelector } from 'react-redux';
+
+import { acquireDevice, selectSelectedDevice } from '@suite-common/wallet-core';
+
+export const useInitiateThpConnection = () => {
```

Comment:
Nit: It seems weird that there is same name of a hook in `@suite-native/module-device-onboarding` and both do different things. Can we apply better name for either of them to avoid duplicated name?

Reply:
Any suggestions? It does different things because each module has different flows.